File: 백준/Bronze/1546. 평균/평균.py
N = int(input()) # 과목의 개수
scores = list(map(int, input().split())) # 시험 점수
new_scores = 0

max_score = max(scores)

for score in scores:
    new_scores += (score/max_score * 100)

ans = new_scores/N

print(ans)

File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        double maxScore = 0;
        double totalScore = 0d;
        double average;
        double[] scores = new double[n];

        for (int i = 0; i < n; i++) {
            double score = sc.nextDouble();
            scores[i] = score;

            if (maxScore < score) {
                maxScore = score;
            }
        }

        for (double score : scores) {
            totalScore += score / maxScore * 100;
        }

        average = totalScore / n;
        System.out.println(average);
    }
}
